### ISSUE TITLE
Support. How to install firejail on Debian so that it does not interfere with any programs?

### ISSUE THREAD

**Issue by Salim9304** [netblue30/firejail#4011]
I have installed firejail, firejail-profiles, firetools.
I would like to run only a couple of programs in the sandbox. But Firejail intervened in all programs and changed their settings (for example, Pluma text editor was damaged, etc.).
I also could not remove the shortcut of any application from the firetools GUI.
All I need is to selectively run applications through the sandbox, but firejail takes over the system without asking me.

**Comment by rusty-snake** [comment]
> How to install firejail on Debian so that it does not interfere with any programs?

Do not run `sudo firecfg` after the installation. If you did, run `sudo firecfg --clean` to undo.

**Comment by Salim9304** [comment]
> > How to install firejail on Debian so that it does not interfere with any programs?
> 
> Do not run `sudo firecfg` after the installation. If you did, run `sudo firecfg --clean` to undo.

I didn’t do sudo firecfg (I didn’t do anything at all except setup)
I removed firejail (`apt-get purge` ...) completely.
I have installed firejail
`apt-get install firejail = 0.9.58.2-2 + deb10u1 firejail-profiles = 0.9.58.2-2 + deb10u1 firetools = 0.9.58-1`

After rebooting the system, firejail took control of all applications again.
I did `sudo firecfg --clean` and it helped.
Thank.
